Python/Product/Uwp/Interpreter/PythonUwpInterpreterFactoryProvider.cs

```diff
@@ -23,39 +25,68 @@ namespace Microsoft.PythonTools.Uwp.Interpreter {
     class PythonUwpInterpreterFactoryProvider : IPythonInterpreterFactoryProvider {
         private HashSet<IPythonInterpreterFactory> _factories = null;
 
-        public const string DefaultVersion = "3.5";
-
         public event EventHandler InterpreterFactoriesChanged;
 
         public PythonUwpInterpreterFactoryProvider() {
-            PythonVersion = DefaultVersion;
-        }
-
-        public string PythonVersion {
-            get; set;
         }
 
         private void DiscoverFactories() {
             if (_factories == null) {
-                var defaultConfig = new InterpreterConfiguration(
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    ProcessorArchitecture.None,
-                    new Version(PythonVersion),
-                    InterpreterUIMode.CannotBeDefault
-                );
+                var userSdkInstallDir = new DirectoryInfo(Environment.ExpandEnvironmentVariables(@"%LOCALAPPDATA%\Microsoft\VisualStudio\%VISUALSTUDIOVERSION%Exp\Extensions\Microsoft\Python UWP"));
+                var sdkInstallDir = new DirectoryInfo(Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0\ExtensionSDKs\Python UWP"));
 
                 _factories = new HashSet<IPythonInterpreterFactory>();
```

Comment:
Better to do this in the constructor now. Maybe make it a dictionary based on the targets file path so we can avoid re-adding factories every time it's called. (Can't remember off the top of my head whether we require IPythonInterpreterFactory identity to be preserved - I think we do, which means you need to avoid constructing new ones on each call if at all possible.)